providers/summary/clearlydefined.js

```diff
@@ -132,20 +135,35 @@ class ClearlyDescribedSummarizer {
     let licenses = licenseUrls.map(extractLicenseFromLicenseUrl).filter(x => x)
     if (!licenses.length) licenses = licenseNames.map(x => SPDX.lookupByName(x) || x).filter(x => x)
     if (licenses.length) setIfValue(result, 'licensed.declared', SPDX.normalize(licenses.join(' OR ')))
+    const registryUrl = buildRegistryUrl(coordinates)
+    const versionUrl = buildVersionUrl(coordinates)
+    const downloadUrl = buildDownloadUrl(coordinates)
+    if (get(result, 'described'))
+      setIfValue(result, 'described.urls', { registry: registryUrl, version: versionUrl, download: downloadUrl })
```

Comment:
If the consts happened to have matching names (e.g., `registry` rather than `registryUrl`) then this could just be `{ registry, version, download }`

Reply:
Right, we can save some code doing this

Reply:
mostly just cleaner. The real code saving comes from removing all the duplicate code.